code/core/src/manager/components/sidebar/TestingModule.tsx

```diff
@@ -199,9 +199,13 @@ export const TestingModule = ({
           <Content ref={contentRef}>
             {testProviders.map((state) => {
               const { render: Render } = state;
-              return (
+              return Render ? (
+                <Fragment key={state.id}>
+                  <Render {...state} />
+                </Fragment>
+              ) : (
                 <TestProvider key={state.id} data-module-id={state.id}>
-                  {Render ? <Render {...state} /> : <LegacyRender {...state} />}
+                  <LegacyRender {...state} />
```

Comment:
@ghengeveld Do you think we could drop the legacy render completely?

Reply:
It's still being used by VTA right now. I haven't updated the VTA yet and people will be using the old version for a while.